python/ray/utils.py

```diff
@@ -66,6 +66,7 @@ def push_error_to_driver(worker,
         data: This should be a dictionary mapping strings to strings. It
             will be serialized with json and stored in Redis.
     """
+    return
```

Comment:
this should be removed

Reply:
Removing this will cause a significant amount of printing to be done to the console. This is not the correct solution, but I would like to discuss what should be done for this (a config for turning off this feature).

Reply:
+1 to keep this as a flag.

Then if we push the error, we should say "by the way to turn this off set ray.init(verbose_failures=False) or something".

RLlib and Tune can turn this off by default since this is not helpful for their execution model.

Reply:
We can have an option for suppressing errors, but that should be a separate PR. Errors need to be displayed by default.